src/targets/chrome/add-loki-session-marker.js

```diff
@@ -0,0 +1,4 @@
+const addLokiSessionMarker = window =>
+  window.document.querySelector(':root').setAttribute('loki-tests', true);
```

Comment:
```suggestion
  window.document.querySelector(':root').setAttribute('loki-test', true);
```

Somehow feels this should be in singularis.

Reply:
updated